presto-docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -5,6 +5,23 @@ Secure Internal Communication
 The Presto cluster can be configured to use secured communication. Communication
 between Presto nodes can be secured with SSL/TLS.
 
+Internal Authentication
+-----------------------
+
+Requests between Presto nodes are authenticated using a shared secret.  The shared
```

Comment:
Off topic question ... when you talk about nodes you mean all controllers and workers in all clusters right? Or just all workers..

Reply:
"all nodes" being every Presto process... coordinators and workers

Reply:
Thanks for explanation.